src/test/java/skeleton/ListSteps.java

```diff
@@ -0,0 +1,53 @@
+package skeleton;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by korobitsyna on 5/24/15.
+ */
+public class ListSteps {
+
+    private ArrayList<String> testList;
```

Comment:
FYI
Here, usage of ArrayList as an argument type might be correct because we are testing this Java's class.
Usually it's always better to use interface List rather than ArrayList:
private List<String> testLst;